client/blocks/reader-author-link/index.jsx

```diff
@@ -29,7 +29,7 @@ const ReaderAuthorLink = ( { author, post, siteUrl, children } ) => {
 	}
 
 	return (
-		<ExternalLink className="reader-author-link" href={ linkUrl } target="_blank" onClick={ recordAuthorClick }>
+		<ExternalLink className="reader-author-link" href={ linkUrl } target="_blank" rel="noopener noreferrer" onClick={ recordAuthorClick }>
```

Comment:
As I suspected, the eslint rule is a bit of a blunt instrument. If we specify `target="_blank"` on an `ExternalLink`, the rule complains even though we are setting `rel` in the underlying component.

I've worked around the handful of cases with

`eslint-disable react/jsx-no-target-blank`

for the moment. Another way around this would be to introduce a new prop on `ExternalLink` (openInNewWindow or similar) that applies `target` inside the component.